CODING_STYLE.md

```diff
@@ -0,0 +1,172 @@
+# Zilliqa Coding Style
```

Comment:
Some bullet points have a trailing period, the others don't. Please ensure consistency across this file and the others.